webapp/components/search_results.jsx

```diff
@@ -175,7 +188,17 @@ export default class SearchResults extends React.Component {
 
         var ctls = null;
 
-        if (this.props.isFlaggedPosts && noResults) {
+        if (this.state.loading) {
+            ctls =
+            (
+                <div className='sidebar--right__subheader loading'>
+                    <img
+                        className='spinner'
+                        src={loadingGif}
+                    />
+                </div>
```

Comment:
This causes a spinner to be rendered in place of existing search results (if any)